lib/msal-common/src/request/BaseAuthRequest.ts

```diff
@@ -17,6 +18,7 @@ export type BaseAuthRequest = {
     correlationId: string;
     scopes: Array<string>;
     claims?: string;
+    clientClaims?: string;
```

Comment:
`clientClaims` may get confused with `claims`, maybe use `requestClientClaims` or `shrClientClaims`?

Reply:
The claim name comes from the cross-platform spec, so I think for parity's sake we should keep `clientClaims`. Still open to discussing this further.

Reply:
I agree, can we call this something like `shrClaims`?

Reply:
I meant the variable on the request object. I understand other platforms are also implementing this feature but I don't think that should stop us from using descriptive naming.

Reply:
Changed to `shrClaims`